routes/produce.js

```diff
@@ -96,4 +282,201 @@ router.get(
   }
 );
 
+//Filter for Pricepage
+router.get(
+  '/pricepage/filter/:range',
+  require('connect-ensure-login').ensureLoggedIn({ redirectTo: '/app/auth/login' }),
+  function (req, res, next) {
+    if (req.user.role === ROLES.Admin && ROLES.Superuser) {
```

Comment:
This should be admin or superuser i.e. `(req.user.role === ROLES.Admin || ROLES.Superuser)`